samples/SmokeTest/SmokeTest/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Azure.Messaging.EventHubs;
+using System;
+using System.Reflection.Metadata;
+using System.Threading.Tasks;
+
+namespace SmokeTest
+{
+    class Program
+    {
+        static async Task<int> Main(string[] args)
+        {                        
+            Console.WriteLine("SMOKE TEST FOR TRACK 2 LIBRARIES");
+
+            var keyVault = new KeyVaultTest("SmokeTestSecret", "smokeTestValue", Environment.GetEnvironmentVariable("DIR_TENANT_ID"), Environment.GetEnvironmentVariable("APP_CLIENT_ID"), Environment.GetEnvironmentVariable("CLIENT_SECRET"), Environment.GetEnvironmentVariable("KEY_VAULT_URI"));
```

Comment:
We should turn these into static method calls and move the configuration into the individual sample file.